sources/models/src/vmware-dev/mod.rs

```diff
@@ -0,0 +1,21 @@
+use model_derive::model;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+use crate::modeled_types::Identifier;
+use crate::{
+    AwsSettings, ContainerImage, KernelSettings, NetworkSettings, NtpSettings, UpdatesSettings,
+};
+
+// Note: we have to use 'rename' here because the top-level Settings structure is the only one
+// that uses its name in serialization; internal structures use the field name that points to it
+#[model(rename = "settings", impl_default = true)]
+struct Settings {
+    motd: String,
+    updates: UpdatesSettings,
+    host_containers: HashMap<Identifier, ContainerImage>,
+    ntp: NtpSettings,
+    network: NetworkSettings,
+    kernel: KernelSettings,
+    aws: AwsSettings,
```

Comment:
might be worth a comment that we intend to remove this setting

Reply:
Per some offline conversation - we removed this completely.  :)

Reply:
I'm hesitant to support merging a new variant that's known to be broken.  We're definitely working toward removing this crutch, and it's easy to test what happens if we remove it...

Reply:
I can see where you're coming from, but this variant is to facilitate development.  It's hard to support fixing the variant if there's not a variant to build. :)

Reply:
> It's hard to support fixing the variant if there's not a variant to build.

I don't understand.  The new variant would be available, would have `settings.aws` so it's not broken, and development could continue toward removing `settings.aws`.